### PR TITLE
Integrate MongoDB with Django

### LINKED ISSUE #2: Database integration
Marking it up as in issue until we merge. Feel free to list blockers here.

### PR DESCRIPTION
**Pull request by QEDK**
Fixes #2

### CODE REVIEW

File: app/app/settings.py [hunk 1]
```diff
@@ -50,6 +50,16 @@
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
+	'app.corsMiddleware.cors_middleware',
+	'corsheaders.middleware.CorsMiddleware'
```
Comment (QEDK) [inline app/app/settings.py:54]: @routrohan this needs to be in requirements.txt

File: app/app/settings.py [hunk 2]
```diff
@@ -29,7 +29,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['localhost']
```
Comment (QEDK) [inline app/app/settings.py:32]: This is duplicated below.

File: app/app/settings.py [hunk 3]
```diff
@@ -50,6 +50,16 @@
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
+	'app.corsMiddleware.cors_middleware',
+	'corsheaders.middleware.CorsMiddleware'
+]
+CORS_ORIGIN_ALLOW_ALL = True
+ALLOWED_HOSTS = ['*']
+
+CORS_ORIGIN_WHITELIST = [
```
Comment (QEDK) [inline app/app/settings.py:59]: This placeholder needs to be modified.

**Review by QEDK — COMMENTED** [review]
@routrohan

File: app/app/settings.py (hunk 1 above)
Comment (routrohan) [reply app/app/settings.py:54]: I followed [this](https://stackoverflow.com/questions/44037474/cors-error-while-consuming-calling-rest-api-with-react) and it worked for me without any issues
Just add `pip install django-cors-headers` to the `requirements.txt` and we're good to go

File: app/app/settings.py (hunk 2 above)
Comment (routrohan) [reply app/app/settings.py:32]: yeah let's just keep `ALLOWED_HOSTS = ['localhost']`

File: app/app/settings.py (hunk 3 above)
Comment (routrohan) [reply app/app/settings.py:59]: I think this works fine 
See anyway we're not going to deal with CORS once we've hosted the website so it's fine

**Pull request merged**